src/components/footer/footer.js

```diff
@@ -1,5 +1,8 @@
 import React from 'react';
-import Twitter from '../../IconComponents/Twitter'
+import Twitter from '../../IconComponents/Twitter';
+import Facebook from '../../IconComponents/Facebook';
+import Instagram from '../../IconComponents/Instagram';
+import Youtube from '../../IconComponents/Youtube';
 
```

Comment:
@nicholasmiller1 Personally, I export files like these through `IconComponents/index.js`, so you can just do:

`import { Twitter, Facebook, Instagram, Youtube } from '../../IconComponents';`